server/src/main/java/org/candlepin/pinsetter/tasks/UniqueByEntityJob.java

```diff
@@ -66,8 +66,8 @@ public static boolean isSchedulable(JobCurator jobCurator, JobStatus status) {
             return false;
         }
 
-        long running = jobCurator.findNumRunningByClassAndTarget(
-            status.getTargetId(), jobClass);
-        return running == 0;  // We can start the job if there are 0 like it running
+        JobStatus nextJob = jobCurator.getNextByClassAndTarget(status.getTargetId(),
+            HypervisorUpdateJob.class);
```

Comment:
SHouldn't this be UniqueByEntityJob?

Reply:
Not unless we want not to queue incoming reports